### PR TITLE
Do not let a non-admin get nil org. They can do stuff to things that is ...

### PR DESCRIPTION
**Pull request by jmontleon**
...not theirs

### CODE REVIEW

File: app/views/home/_organization_dropdown.rhtml [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <%= link_to "New Organization", new_organization_path, :class => "btn btn-primary" %>
       </div>
     <% else %>
-      <% if orgs_count > 1 || User.current.admin? %>
+      <% if orgs_count > 1 && User.current.admin? %>
```
Comment (skottler) [inline app/views/home/_organization_dropdown.rhtml:16]: Maybe we should create the concept of "All my organizations" which would displayed everything across all the organizations to which the current user has access. The admin wouldn't see this, they'd just have the existing "All organizations" choice.

**Comment by jmontleon** [comment]
Yes, that would be great, though a little more challenging. Right now it seems the only way to see all is by setting the Org to nil.

File: app/views/home/_organization_dropdown.rhtml (hunk 1 above)
Comment (ohadlevy) [inline app/views/home/_organization_dropdown.rhtml:16]: afaik we already support this! @abenari please review?

**Comment by abenari** [comment]
issue already solved in the commit that re implemented the org switcher.

**Pull request closed without merging**